wagtail/admin/forms/pages.py

```diff
@@ -18,7 +20,17 @@ def __init__(self, *args, **kwargs):
         can_publish = kwargs.pop("can_publish")
         super().__init__(*args, **kwargs)
         self.fields["new_title"] = forms.CharField(
-            initial=self.page.title, label=_("New title")
+            initial=self.page.title, label=_("New title"),
+            widget=forms.TextInput(
+                attrs={
+                    "placeholder": format_lazy(
+                        "{title}*", title=gettext_lazy("Page title")
+                    ),
+                    "data-controller": "w-sync",
+                    "data-w-sync-target-value": "body:not(.page-is-live) #id_slug",
+                    "data-action": "focus->w-sync#check blur->w-sync#apply change->w-sync#clear",
+                }
+            ),
         )
```

Comment:
I recommend we ignore this on the copy page, it's not current behaviour and actually makes less sense for a page copy to have this behaviour. 

Maybe it can be added as an enhancement later.

Reply:
Alright
During the process of writing the test, disconnect method always brings up errors whenever I run the test. It makes the test not to run at all.